lib/applicants/schema.graphql

```diff
@@ -75,3 +76,19 @@ input UpdateApplicantInput {
 type UpdateApplicantResult {
   ok: Boolean!
 }
+
+input ApplicantsFilter {
+  order: [[String]]
+  permitStatus: PermitStatus
+  userStatus: UserStatus
+  expiryDateRangeFrom: Date
+  expiryDateRangeTo: Date
+  search: String
+  limit: Int
+  offset: Int
+}
+
+type QueryApplicantsResult {
+  result: [Applicant]!
```

Comment:
```suggestion
  result: [Applicant!]!
```
since every `Applicant` must be non-null

Reply:
true i guess there should always be applicants in the db